.github/workflows/automate-team-review-assignment.yml

```
@@ -63,24 +61,35 @@ jobs:
-          uses: shufo/auto-assign-reviewer-by-files@f5f3db9ef06bd72ab6978996988c6462cbdaabf6
+          uses: ./.github/actions/assign-reviewers
           with:
-              config: '.github/project-community-pr-assigner.yml'
-              token: ${{ secrets.PR_ASSIGN_TOKEN }}
+              mode: changed-files
+              config: .github/project-community-pr-assigner.json
+              github-token: ${{ secrets.PR_ASSIGN_TOKEN }}
 
         - name: Assign reviewers for a teams PR
           if: ${{ steps.check.outputs.contributor == 'no' && github.event.pull_request.draft == false && ! contains( github.event.pull_request.labels.*.name, 'Release' ) && ( github.event.pull_request.base.ref == 'trunk' || startsWith( github.event.pull_request.base.ref, 'release/' ) ) }}
           continue-on-error: ${{ ( github.event.pull_request.head.repo.fork && 'true' ) || 'false' }}
```

actionlint (1.7.12)

[error] type of expression must be bool but found type string

(expression)